src/main/java/frc/robot/Constants.java

```diff
@@ -56,10 +56,10 @@
 
         public static final double TRANSLATIONAL_FRICTION = 0.0205;
 
-        public static final double FRONT_LEFT_ROTATION_OFFSET = 0;
-        public static final double FRONT_RIGHT_ROTATION_OFFSET = 0;
-        public static final double BACK_LEFT_ROTATION_OFFSET = 0;
-        public static final double BACK_RIGHT_ROTATION_OFFSET = 0;
+        public static final double FRONT_LEFT_ROTATION_OFFSET = 2.75 - (Math.PI/5);
```

Comment:
why - (Math.PI/5)

Reply:
because that's how it worked. It needs to be re-tuned today anyways.